app/src/main/java/com/example/dispatcher/presentation/search/view/SearchFragment.kt

```diff
@@ -0,0 +1,113 @@
+package com.example.dispatcher.presentation.search.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.dispatcher.R
+import com.example.dispatcher.databinding.FragmentSearchBinding
+import com.example.dispatcher.presentation.homepage.view.adapter.ArticleAdapter
+import com.example.dispatcher.presentation.homepage.view.adapter.TopSpacingItemDecoration
+import com.example.dispatcher.presentation.homepage.viewModel.ArticlesViewModel
+import com.example.dispatcher.presentation.main.MainViewModel
+import com.example.dispatcher.presentation.search.view.adapter.SearchHistoryAdapter
+import com.example.dispatcher.presentation.search.viewModel.SearchViewModel
+
+class SearchFragment : Fragment(R.layout.fragment_search) {
```

Comment:
i found a bug, when i search the first query and get no result - i see the empty state image. 
then i want to search something else - is see the first search in the recent searches, but when i search the second query - i get no indication for the search